hack/verify-vendor.sh

```diff
@@ -1,12 +1,9 @@
 #!/bin/sh
 
 if [ "$IS_CONTAINER" != "" ]; then
-  if [ ! "$(command -v dep >/dev/null)" ]; then
-    go get -u github.com/golang/dep/cmd/dep
-  fi
-
-  dep check
-  dep ensure
+  export GO111MODULE=on
+  GOPROXY=https://proxy.golang.org go mod vendor
```

Comment:
what is the GOPROXY needed for exactly? Just asking because this wasn't set with dep.

Reply:
fair question: see e.g. https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/2745/pull-ci-openshift-installer-master-images/7117#1:build-log.txt%3A279

```
 go: modernc.org/strutil@v1.0.0: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /go/pkg/mod/cache/vcs/f48599000415ab70c2f95dc7528c585820ed37ee15d27040a550487e83a41748: exit status 128:
	error: RPC failed; result=22, HTTP code = 404
	fatal: The remote end hung up unexpectedly 
```

I believe this domain forwards to gitlab.com, and it worked locally. Dunno why this fails in Prow. They may have blacklisted our IPs?
@vrutkovs pointed me to the fix, thanks again!

Reply:
Go 1.13 defaults GOPROXY to to `https://proxy.golang.org,direct`. Removed again.